lib/telnet.c

```diff
@@ -1439,6 +1334,7 @@ static CURLcode telnet_do(struct connectdata *conn, bool *done)
     DWORD waitret = WaitForMultipleObjects(obj_count, objs,
                                            FALSE, wait_timeout);
     switch(waitret) {
+
```

Comment:
Minor nit: Not sure if the extra line was intentionally added above.

Reply:
I added this on purpose, because there is also a blank line at the end of the statement to separate the two `}` which appear on the same line due to the case..break statements not being intended.